bin/launch.rb

```diff
@@ -40,11 +40,11 @@
 end
 
 # Need to initialize the log like this once, because otherwise it only becomes available after the Sinatra app has received a request...
-::SemanticLogger.add_appender(file_name: "#{RubyDeps.environment}.log")
+::SemanticLogger.add_appender(file_name: "#{CuttingEdge.environment}.log")
 
-RubyDeps.set(:repositories, repositories)
-RubyDeps.set(:store, store)
-RubyDeps.set(:enable_logging, true)
+CuttingEdge.set(:repositories, repositories)
```

Comment:
Perhaps this should be `CuttingEdge::App`?

Reply:
Done.